corehq/tests/noseplugins/uniformresult.py

```diff
@@ -26,7 +26,12 @@ def uniform_description(test):
     if isinstance(test, type):
         return "%s:%s" % (test.__module__, test.__name__)
     if isinstance(test, FunctionTestCase):
-        return str(test)
+        descriptor = test.descriptor or test.test
+        return "%s:%s args %s" % (
```

Comment:
Nitpick: I'd change this to `return "%s:%s %s" % (` – shorter is better, and there's not much ambiguity when the args tuple is surrounded by parens anyway.